acsoo/tag_requirements.py

```diff
@@ -128,7 +128,7 @@ def do_tag_requirements(config, force, src, requirement, yes):
             check_call(['git', 'tag'] + force_cmd + [eggtag, sha])
             try:
                 check_call(['git', 'push'] + force_cmd + [push_url, eggtag])
-            except:
+            except Exception:
```

Comment:
In fact here the bare except is adequate, because we want to do cleanup the tag if someone does ctrl-c during the push.